### PR TITLE
add complete set of datastore view rules

### PR DESCRIPTION
**Pull request by ben-lerch**
opening this to track progress.

### CODE REVIEW

File: pbsmrtpipe/pb_pipelines/pb_pipeline_view_rules.py [hunk 1]
```diff
@@ -38,6 +38,11 @@ def wrapper(*args, **kwds):
             return func(*args, **kwds)
     return deco_wrapper
 
+def _log_file_rules():
```
Comment (natechols) [inline pbsmrtpipe/pb_pipelines/pb_pipeline_view_rules.py]: Do we really want to hide log files?

Comment (ben-lerch) [inline pbsmrtpipe/pb_pipelines/pb_pipeline_view_rules.py]: I set isHidden to false, so it produces:

        {
            "description": null,
            "fileTypeId": "PacBio.FileTypes.log",
            "isHidden": false,
            "name": null,
            "sourceId": "pbsmrtpipe::master.log"
        },
        {
            "description": null,
            "fileTypeId": "PacBio.FileTypes.log",
            "isHidden": false,
            "name": null,
            "sourceId": "pbsmrtpipe::pbsmrtpipe.log"
        },

Comment (natechols) [inline pbsmrtpipe/pb_pipelines/pb_pipeline_view_rules.py]: You don't need to define view rules for every output file - for now we just want to cover the ones that need to be hidden.

Comment (ben-lerch) [inline pbsmrtpipe/pb_pipelines/pb_pipeline_view_rules.py]: ah ok, good to know :)

**Pull request merged**